packages/story-editor/src/components/keyboardShortcutsMenu/shortcutMenuSection.js

```diff
@@ -79,10 +79,10 @@ function ShortcutMenuSection({ title, commands }) {
       <List title={title}>
         {commands.map(({ label, shortcut }) => (
           <ListRow key={label}>
+            <ShortcutLabel keys={shortcut} alignment={'left'} />
```

Comment:
This puts the `<dt>` element _**after**_ the `<dd>` element so text selection works properly. If reordering the elements like this is invalid markup and causes accessibility or other issues, another element can be used (`<div>` or `<span>`) and appropriate `aria-*` attributes can be added if needed.